frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/sc-node-list.tsx

```diff
@@ -59,22 +120,13 @@ const getRows: GetRows = ({ componentProps }) => {
   return rows;
 };
 
-const AttachedDevicesNodeTable: React.FC<NodeTableProps> = (props) => {
-  const { t } = useTranslation();
-
-  const { data, customData } = props;
-  const { filteredNodes, nodes = [], setNodes } = customData;
-  const tableData: NodeKind[] = data.filter(
-    (node: NodeKind) =>
-      filteredNodes.includes(getName(node)) ||
-      filteredNodes.includes(node.metadata.labels?.['kubernetes.io/hostname']),
-  );
+type AttachedDevicesNodeTableProps = NodeTableProps<NodeKindWithLoading> & {
+  disableCustomLoading?: boolean;
+};
 
-  React.useEffect(() => {
-    if (setNodes && !_.isEqual(tableData, nodes)) {
-      setNodes(tableData);
-    }
-  }, [tableData, setNodes, nodes, filteredNodes]);
+const AttachedDevicesNodeTable: React.FC<AttachedDevicesNodeTableProps> = (props) => {
+  const { t } = useTranslation();
+  const { disableCustomLoading = false } = props;
```

Comment:
```suggestion
const AttachedDevicesNodeTable: React.FC<AttachedDevicesNodeTableProps> = ({ disableCustomLoading = false, data, loaded }) => {
  const { t } = useTranslation();
```

Reply:
No need to do it. I can directly pass props.